src/test/HST/ApplicationTests.hs

```diff
@@ -92,37 +92,37 @@ tests = do
       ++ "id :: a -> a\n"
       ++ "id x | otherwise = x"
     let mod2 = evalPM (processModule mod1) defaultState
-    expected <-
-      parseTestModule
-      $  "module A where\n"
-      ++ "id :: a -> a\n"
-      ++ "id a0 = let a2 = undefined\n"
-      ++ "            a1 = case a0 of\n"
-      ++ "              a3 -> if otherwise then a3"
-      ++ "                    else a2\n"
-      ++ "         in a1"
+    expected <- parseTestModule $ unlines
+      [ "module A where\n"
+      , "id :: a -> a\n"
+      , "id a0 = let a2 = undefined"
+      , "            a1 = case a0 of"
+      , "              a3 -> if otherwise then a3"
+      , "                    else a2"
+      , "         in a1"
+      ]
     mod2 `prettyShouldBe` expected
   it "should accept a more complex guarded function" $ do
-    mod1 <-
-      parseTestModule
-      $  "module A where\n"
-      ++ "useless :: (a -> Bool) -> a -> a -> a\n"
-      ++ "useless p x y | p x       = x\n"
-      ++ "              | otherwise = y"
+    mod1 <- parseTestModule $ unlines
+      [ "module A where\n"
+      , "useless :: (a -> Bool) -> a -> a -> a"
+      , "useless p x y | p x       = x"
+      , "              | otherwise = y"
+      ]
     let mod2 = evalPM (processModule mod1) defaultState
-    expected <-
-      parseTestModule
-      $  "module A where\n"
-      ++ "useless :: (a -> Bool) -> a -> a -> a\n"
-      ++ "useless a0 a1 a2 = let\n"
-      ++ "                     a4 = undefined\n"
-      ++ "                     a3 = case a0 of\n"
-      ++ "                       a5 -> case a1 of\n"
-      ++ "                        a6 -> case a2 of\n"
-      ++ "                         a7 -> if a5 a6 then a6\n"
-      ++ "                               else if otherwise then a7\n"
-      ++ "                                    else a4\n"
-      ++ "                   in a3"
+    expected <- parseTestModule $ unlines
+      [ "module A where\n"
+      , "useless :: (a -> Bool) -> a -> a -> a"
+      , "useless a0 a1 a2 = let"
+      , "                     a4 = undefined"
+      , "                     a3 = case a0 of"
+      , "                       a5 -> case a1 of"
+      , "                        a6 -> case a2 of"
+      , "                         a7 -> if a5 a6 then a6"
+      , "                               else if otherwise then a7"
+      , "                                    else a4"
+      , "                   in a3"
```

Comment:
This one could be indented differently as well, for example like this:
```suggestion
      , "useless a0 a1 a2 ="
      , "  let a4 = undefined"
      , "      a3 = case a0 of"
      , "        a5 -> case a1 of"
      , "          a6 -> case a2 of"
      , "            a7 -> if a5 a6 then a6"
      , "                           else if otherwise then a7"
      , "                                             else a4"
      , "  in a3"
```
There are different solutions, of course, but sub blocks should be indented two spaces further than the outer block and `then` and `else` should be underneath each other, at least if the `then`- and `else`-expressions are equally long.